python_basics_lists_dictionaries/python_basics_lists_dictionaries.md

```diff
@@ -0,0 +1,453 @@
+    print(produce)
+
+    </script>
+    </lia-keep>
+    </div>
+
+  * `del` deletes objects, and can be used as another way to remove elements of a list by index. The syntax will be a little different, because it is a [**keyword**](https://realpython.com/python-keywords/), not a list method. It is useful because it allows you to delete list elements in a **slice**. Slicing uses subsetting notation to indicate where the slice begins and ends. The first index listed is **inclusive**, or included in the slice; this is followed by a colon; the second index listed is **exclusive**, or excluded from the slice; finally, these numbers are surrounded by square brackets.
+
+    @sage
+    <div class="python_link">
+    <lia-keep>
+    <script type="text/x-sage">
+
+    del produce[1:3]
+print(produce)
+
+    </script>
+    </lia-keep>
+    </div>
+
+  * `.clear()` removes everything from the list.
+
+    @sage
+    <div class="python_link">
+    <lia-keep>
+    <script type="text/x-sage">
+
+    produce.clear()
+print(produce)
+
+    </script>
+    </lia-keep>
+    </div>
+
+  * `.append()` adds an element to the end of a list.
+
+    @sage
+    <div class="python_link">
+    <lia-keep>
+    <script type="text/x-sage">
+
+    produce.append("pear")
+print(produce)
+
+    </script>
+    </lia-keep>
+    </div>
+
+  * `.insert()` adds an element to a specific position.
+
+    @sage
+    <div class="python_link">
+    <lia-keep>
+    <script type="text/x-sage">
+
+    produce.insert(0, "kale")
+print(produce)
+
+    </script>
+    </lia-keep>
+    </div>
+
+  * `.extend()` adds another list to your list (or another iterable object, but don't worry about that for now).
+
+    @sage
+    <div class="python_link">
+    <lia-keep>
+    <script type="text/x-sage">
+    berries= ["strawberry", "blueberry", "raspberries"]
+produce.extend(berries)
+print(produce)
+
+    </script>
+    </lia-keep>
+    </div>
+
+
+You'll notice that with each step, our list `produce` has been permanently changed-- in order to "start over" with the original list, we simply need to re-run the first code chunk.
+
+<div class = "help">
+<b style="color: rgb(var(--color-highlight));">Troubleshooting help</b><br>
+
+The code cells above are linked, meaning that all of the cells "remember" the code that has been run previously (as they would be in a [notebook](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/demystifying_python/demystifying_python.md#jupyter-notebooks)). Because of this, if you run them out of order, you can get unexpected results or even errors! If things aren't working like they should, re-run the first code cell on the page or hit your refresh button.
+
+</div>
+
+### Quiz: Lists
+
+1. Which of the following is FALSE about lists? Select all that apply.
+
+    [( )] Lists are changeable; items can be added, removed, or replaced after the list is created.
+    [( )] Lists can be assigned to variables.
```

Comment:
The previous page assigned the list to the variable `produce`, but didn't explicitly say that was what was happening. I suggest adding an explanation in the answer to this question now that the list section no longer immediately follows the variables section. Something like
> "Even though we didn't explicitly discuss it, the list on the previous page was assigned to the variable `produce`."